src/vs/workbench/contrib/webviewPanel/browser/webviewEditor.ts

```diff
@@ -189,7 +188,7 @@ export class WebviewEditor extends EditorPane {
 			containsGroup: (group) => this.group?.id === group.id
 		}));
 
-		this._webviewVisibleDisposables.add(new WebviewWindowDragMonitor(() => this.webview));
+		this._webviewVisibleDisposables.add(new WebviewWindowDragMonitor(input.webview.codeWindow, () => this.webview));
```

Comment:
Instead of getting the codeWindow from the webview here, could we get the window associated with the editor? Then we may not need to expose `webview.codeWindow`

Reply:
This is a good catch.